spec/mailers/candidate_mailer_spec.rb

```diff
@@ -558,7 +558,8 @@
         'a mail with subject and content',
         'Duplicate application detected',
         'greeting' => 'Dear Fred,',
-        'details' => 'We’ve noticed that you’ve started multiple applications on the DfE’s Apply for teacher training.',
+        'details' => 'You’ve created more than one account on Apply for teacher training, and you have more than 3 applications in progress.',
```

Comment:
Is this necessarily true?

Reply:
No, this content has been removed, I haven't updated the test yet